ee/clickhouse/views/test/test_clickhouse_funnel_person.py

```diff
@@ -1,33 +1,49 @@
 import json
+from uuid import uuid4
```

Comment:
General question about how these tests are written:

Are the `views` tests testing the API format / connection?

While the `queries` test test the actual query result?

Reply:
yep